pkg/reconciler/revision/background_test.go

```diff
@@ -224,11 +224,11 @@ func TestRateLimitGlobal(t *testing.T) {
 		subject.Resolve(logger, rev(name, name+"img1", name+"img2"), k8schain.Options{ServiceAccountName: "san"}, sets.NewString("skip"), 0)
 	}
 
-	// Rate limit base rate in this test is 1 second, so this should give time for at most 1 resolve.
+	// No rate limiting will apply on the first try; 3 containers * 5 revisions == 15 resolves
 	time.Sleep(500 * time.Millisecond)
 
-	if r := resolves.Load(); r > 1 {
-		t.Fatalf("Expected resolves to be rate limited, but was called %d times", r)
+	if r := resolves.Load(); r != 15 {
```

Comment:
Is there any chance `r != 15` could flake based on any timings? Specifically is this test deterministic?

Reply:
Theoretically, yes. Taking another glance at it I might even be tempted to remove that test altogether since the delay scenarios are already covered by this following test:
https://github.com/knative/serving/blob/84ea41d9158441dbc19ae37c4b06c903bb82e295/pkg/reconciler/revision/background_test.go#L235

We would also lose no coverage when removing it.
What do you think @dprotaso ?

Reply:
Sure let's remove the test